public/app/ui/pages/search/SearchPage.js

```diff
@@ -45,6 +46,7 @@
                                     return new DynamicRecord(result, this.searchResultFields);
                                 }.bind(this)));
                                 this.displayResults(true);
+                                responsive.update();
```

Comment:
In the search module, an additional call to `responsive.update()` is used here so that when the search results come in (which changes the page view) the footer position is recalculated.  If this line is not here, the footer will stay pinned when the results are displayed, even if the results are longer than the page.